packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -0,0 +1,295 @@
+import React from 'react';
+import styled from 'styled-components';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  C_WHITE,
+  C_POSTBOX,
+  C_SHADOW,
+  C_EBON,
+  C_METAL,
+} from '@bbc/psammead-styles/colours';
+import { string, oneOf, shape, node, bool } from 'prop-types';
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const GEL_SPACING_THREE_QUARTERS = '0.75rem';
+
+const BulletinWrapper = styled.div`
+  background-color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+  position: relative;
+  text-decoration: none;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
+  }
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${C_WHITE};
+    fill: currentColor;
+    margin-right: ${GEL_SPACING};
+    height: ${GEL_SPACING_DBL};
+  }
+`;
+
+const PlayCTA = styled.div.attrs({ 'aria-hidden': true })`
+  background-color: ${({ isLive }) => (isLive ? C_POSTBOX : C_EBON)};
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${C_WHITE};
+  width: 100%;
+  display: block;
+  text-align: center;
+  padding: ${GEL_SPACING_THREE_QUARTERS};
+  vertical-align: middle;
+`;
+
+const StyledSummary = styled.p`
+  ${({ script }) => script && getLongPrimer(script)};
+  ${({ service }) => service && getSansRegular(service)}
+  color: ${C_SHADOW};
+  margin: 0; /* Reset */
+  padding: ${GEL_SPACING} 0 ${GEL_SPACING_DBL};
+`;
+
+const StyledHeading = styled.h3`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => getSansBold(service)}
+  color: ${C_EBON};
+  margin: 0; /* Reset */
+`;
+
+const HeadingLink = styled.a`
```

Comment:
It looks like this `HeadingLink` has the same styles as the `Story Promo Link`. Do you think it would be better to import the link and use it here instead?

Reply:
Also, in the Story Promo, we did in a different way. We export all the elements and then we put them together in the stories and in the Simorgh container, I wonder what is the best approach.

Reply:
Based on discussion in slack, as `RadioBulletin` is mapped over data, the dynamic construction will be done in Simorgh.

Reply:
Ok, I've now created a `psammead-bulletin-container` which extracts the relevant data and composes the `Bulletin` into the form expected in its `TVBulletin` and `RadioBulletin` containers. This should be easy to expand into a `ul` with `li` items in it when we get to that stage, but I don't think that's this task.